### PR TITLE
Readd (old) testbed and ciab docs as 02-iaas/deployment-examples.

### PR DESCRIPTION
**Pull request by garloff**
This is a stop-gap measure until we have the news docs.

### CODE REVIEW

**Comment by garloff** [comment]
Untested and based on a chat exchange with @maxwolfs (THANKS!) ... I will merge this to staging to see how it fares and remove the WIP label if it looks reasonable.

**Review by fkr — APPROVED** [review]
LGTM. Thanks.

**Comment by garloff** [comment]
github.com/osism/cloud-in-a-box/doc does not exist, the build fails. On the other hand, there is reasonable content at https://osism.github.io/docs/advanced-guides/cloud-in-a-box which we should be able to pull in, but I have not found yet, where it comes from exactly. Any insight @berendt, @aeroflydesign?

File: docs.package.json [hunk 1]
```diff
@@ -36,8 +36,8 @@
     "label": "testbed"
   },
   {
-    "repo": "osism/cloud-in-a-box",
-    "source": "doc",
+    "repo": "osism/docs",
```
Comment (berendt) [inline docs.package.json:39]: This is the new repository: https://github.com/osism/osism.github.io

File: docs.package.json [hunk 2]
```diff
@@ -36,8 +36,8 @@
...
+    "source": "advanced-guides/cloud-in-a-box.md",
```
Comment (berendt) [inline docs.package.json:40]: This is the path in the new repository: https://github.com/osism/osism.github.io/tree/main/docs/advanced-guides

**Review by berendt — CHANGES_REQUESTED** [review]

**Comment by berendt** [comment]
We have now moved our docs to a central location to be able to publish in the future as well.

The repository is https://github.com/osism/osism.github.io.

All doc files are inside the docs directory.

The Cloud in a Box guide as well as the Testbed guide are single files that can be integrated in other docs.

File: docs.package.json (hunk 1 above)
Comment (garloff) [reply docs.package.json]: For testbed: Should we use osism/testbed/doc OR better osism/osism.github.io/docs/advanced-guides/testbed.md?

Comment (garloff) [reply docs.package.json]: Ok, we are consuming the osism.github.io pages now.

**Comment by garloff** [comment]
Status:
* With this change, we import ciab and testbed docs from github/osism/osism.github.io advanced-guides into SCS docs now
* The documentation for testbed and c-i-a-b can thus now be found by using the search function
* Any old links to testbed or CiaB docs on docs.scs.community will probably not work
* testbed and CiaB do not show up in the navigation bar on the left side
* I have merged this into the staging branch, so you can see the result in https://docs-staging.scs.community/

**Comment by berendt** [comment]
What are the URLs?

**Review by maxwolfs — APPROVED** [review]
LGTM

**Review by berendt — APPROVED** [review]

**Pull request merged**